src/services/OfficeRnDDataAggregator.tsx

```diff
@@ -31,7 +31,7 @@ export class OfficeRnDDataAggregator {
         timezone: event.timezone,
         room: meetingRoom?.name || 'no meeting room',
         floor: meetingRoom?.floor || 'no floor',
-        team: teamName?.name || memberName?.name || 'no host',
+        host: teamName?.name || memberName?.name || 'no host',
```

Comment:
teamName?.name & memberName?.name use two times "name" inside

It should be better to rename teamName to team and memberName to member